sympy/polys/factortools.py

```diff
@@ -1301,12 +1303,14 @@ def dmp_factor_list(f, u, K0):
             if K0_inexact is None:
                 coeff = coeff/denom
             else:
+                num_factors = 0
                 for i, (f, k) in enumerate(factors):
                     f = dmp_quo_ground(f, denom, u, K0)
```

Comment:
It seems that the wrong coefficient results from this division. Would it be possible to omit this line (and the corresponding line in dup_factor_list) and include `1/denom` (properly converted) among the factors?